ftw/topics/behavior.py

```diff
@@ -3,46 +3,34 @@
 from plone.app.relationfield.event import extract_relations
 from plone.autoform.interfaces import IFormFieldProvider
 from plone.directives.form import widget
-from plone.formwidget.contenttree import MultiContentTreeFieldWidget
-from plone.formwidget.contenttree import ObjPathSourceBinder
-from plone.formwidget.contenttree.utils import closest_content
 from z3c.relationfield.event import _setRelation
-from z3c.relationfield.schema import RelationChoice
 from z3c.relationfield.schema import RelationList
 from zope.component import getMultiAdapter
 from zope.interface import alsoProvides
 from zope.interface import Interface
+from ftw.referencewidget.widget import ReferenceBrowserWidget
```

Comment:
The list of imports seems to be sorted alphabetically. I tend to adhere to the local consistency in this case.